pkg/node/ncl.go

```diff
@@ -15,3 +16,16 @@ func CreateMessageSerDeRegistry() (*ncl.MessageSerDeRegistry, error) {
 	)
 	return reg, err
 }
+
+// computeHeartbeatTopic returns the subject to publish heartbeat messages to.
+// it publishes to the outgoing heartbeat subject of a specific compute node, which
+// the orchestrator subscribes to.
+func computeHeartbeatTopic(nodeID string) string {
+	return fmt.Sprintf("bacalhau.global.compute.%s.out.heartbeat", nodeID)
+}
+
+// orchestratorHeartbeatSubscription returns the subject to subscribe for compute heartbeats.
+// it subscribes for heartbeat messages from all compute nodes
+func orchestratorHeartbeatSubscription() string {
+	return "bacalhau.global.compute.*.out.heartbeat"
+}
```

Comment:
nit: might be better to extract `bacalhau.global.compute.%s.out.heartbeat` to a variable e.g.
```suggestion
const HeartbeatTopicFormat = "bacalhau.global.compute.%s.out.heartbeat"

// computeHeartbeatTopic returns the subject to publish heartbeat messages to.
// it publishes to the outgoing heartbeat subject of a specific compute node, which
// the orchestrator subscribes to.
func computeHeartbeatTopic(nodeID string) string {
	return fmt.Sprintf(HeartbeatTopicFormat, nodeID)
}

// orchestratorHeartbeatSubscription returns the subject to subscribe for compute heartbeats.
// it subscribes for heartbeat messages from all compute nodes
func orchestratorHeartbeatSubscription() string {
	return fmt.Sprintf(HeartbeatTopicFormat, "*")
}
```